### DISCUSSION TITLE
censoring on bark

### DISCUSSION THREAD

**Discussion by raynepaws** [cytrinc/bark-coding#140] [Admin General]
these are allowed on bark:
- crap
- damn
- hell
- piss
- social media links **(exceptions: onlyfans and other p*rn sites)**
- shit
- voice reveal

_(admins: if you disagree with any of these, we can discuss it)_

**Comment by mariocraft987** [comment]
Hell and crap? But not “fuck”

**Reply by raynepaws** [reply]
> But not "fuck"

yes, of course not

**Comment by rubiidev18alt** [comment]
also try to consider edge cases with the replacer, like misspellings of the swears, letetrs replace with cryillic letters, and a lot more others get censored as well

**Comment by raynepaws** [comment]
i think all derogatory terms (c-nt, b-tch, wh-re, etc.) should definitely be censored. how lenient are we going to be about social media platforms, tho? maybe we could add a "you are leaving bark" warning before letting them go

**Reply by raynepaws** [reply]
we'll be more lenient than scratch, but no being mean

**Comment by nmsderp** [comment]
voice reveal seems fine, but face reveal should be censored

**Reply by raynepaws** [reply]
i'll just remove it from the list until we figure that one out

**Reply by raynepaws** [reply]
> Face reveals shouldn't be allowed unless it has censoring or the user is 13+.

i think i can get behind that. what do @nmsderp @mariocraft987 @dumorando @Rubiidev-18 and @Bashamega think of that
sorry for @'ing everyone 🥲

**Comment by Bashamega** [comment]
Maybe we can use Profanity API to filter the text.
